gensim/sklearn_integration/sklearn_wrapper_gensim_ldamodel.py

```diff
@@ -126,6 +126,8 @@ def get_topic_dist(self, bow, minimum_probability=None, minimum_phi_value=None,
     def partial_fit(self, X):
         """
         Train model over X.
+        By default, 'online' mode is used for training the LDA model.
+        Configure `passes` and `update_every` params at init to choose the mode among 'online', 'mini-batch', 'batch'
```

Comment:
I have updated the mode names and also added explicit values to use for each mode.